### PR TITLE
Resolved Readme link to setup dev environment

### LINKED ISSUE #180: Doc: Update link to setup instructions on Readme file.
**Describe the bug**
The previous **BIT Backend Environment Setup** linked to a wiki page that is not longer exist since now we're using the ENV_SETUP_INSTRUCTION.md under .github folder. Therefore the link inside the Readme file need to be updated to reflect this change.

**To Reproduce**
1. Go to bridge-in-tech-backend repository and click on the link to **BIT Development Environment Setup**

2. See an empty wiki page (not the expected behaviour)
Screen Shot 2021-01-22 at 11 04 06 am

3. Now Go to the Wiki page and click on the link to the same page from the side bar

Screen Shot 2021-01-22 at 11 04 33 am

Screen Shot 2021-01-22 at 11 05 20 am

**Expected behavior**
The link on the Readme of repository need to lead to the same page as the wiki side bar does

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina
 - Version: Catalina 10.15.7, Safari 14.0.2

### PR DESCRIPTION
**Pull request by 0xNilesh**
Description
Updated Setup instructions link on Readme file

Fixes #180 

Type of Change:
       Documentation

Code/Quality Assurance Only
        Bug fix (non-breaking change which fixes an issue)

How Has This Been Tested?
        Tried opening link from readme file.

Checklist:
  - [ ] My PR follows the style guidelines of this project
  - [ ] I have made corresponding changes to the documentation
  
Code/Quality Assurance Only
 - [ ] My changes generate no new warnings
 - [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This is the backend client of Bridge-In-Tech.
 Please read our [Contributing Guidelines](.github/CONTRIBUTING.md), [Code of Conduct](.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](.github/REPORTING_GUIDELINES.md) thoroughly.
 
 ## Setup
-To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup) wiki page.
+To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction]( https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/ENV_SETUP_INSTRUCTION.md) wiki page.
```
Comment (isabelcosta) [inline README.md]:
```suggestion
To start contributing to the project, setup the backend environment on your local machine by following the instructions on the [BIT Development Environment Setup Instruction](.github/ENV_SETUP_INSTRUCTION.md) wiki page.
```

you could make the link as a relative path link :)

Comment (0xNilesh) [reply README.md]: @isabelcosta Thanks for reviewing. I have made the suggested changes 😀.

**Review by isabelcosta — APPROVED** [review]
Great work @VectoTec :) Thank you for this contribution!

**Review by mtreacy002 — APPROVED** [review]
Thanks @VectoTec for your contribution and thanks @isabelcosta for suggesting the relative link. I overlooked this on frontend Readme 😂 but will open a first-timers issue for it (along with fixing other typos raised by @VectoTec on this [doc](https://docs.google.com/document/d/12wqYtC8JvhFPqk8-k6T72c9PNZzzch3HW6Zo38jUjw0/edit))

**Pull request merged**